jobs/keepalived/templates/bin/ha_health_check_ctl.erb

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+# Setup common env vars and folders
+source /var/vcap/packages/bosh-helpers/ctl_setup.sh 'keepalived'
+export PID_FILE=${RUN_DIR}/ha_health_check.pid
+export NODE_CMD=/var/vcap/packages/node/bin/node
+
+case $1 in
+
+  start)
+    echo "Starting ha_health_check process..."
```

Comment:
agree during development/debug it might be required, but in the long run the log file will just have the similar entries. Atleast if you can add a date/time into the log it helps.